Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -92,97 +112,270 @@ function sortDialog(dialogs: DialogInfo[]) {
   });
 }
 
-function createItemsAndGroups(
-  dialogs: DialogInfo[],
-  dialogId: string,
-  filter: string
-): { items: any[]; groups: IGroup[] } {
-  let position = 0;
-  const result = dialogs
-    .filter((dialog) => {
-      return dialog.displayName.toLowerCase().includes(filter.toLowerCase());
-    })
-    .reduce(
-      (result: { items: any[]; groups: IGroup[] }, dialog) => {
-        const warningContent = containUnsupportedTriggers(dialog);
-        result.groups.push(createGroupItem(dialog, dialogId, position, warningContent));
-        position += dialog.triggers.length;
-        dialog.triggers.forEach((item, index) => {
-          const warningContent = triggerNotSupported(dialog, item);
-          result.items.push(createItem(item, index, warningContent));
-        });
-        return result;
-      },
-      { items: [], groups: [] }
-    );
-  return result;
-}
-
-interface IProjectTreeProps {
+type BotInProject = {
   dialogs: DialogInfo[];
-  dialogId: string;
-  selected: string;
-  onSelect: (id: string, selected?: string) => void;
+  projectId: string;
+  name: string;
+  isRemote: boolean;
+};
+
+type IProjectTreeProps = {
```

Comment:
nit: Interfaces usually start with I (capital i), types are just PascalCase

Reply:
Fixed (my usual convention is to just name this `Props`, because we already know what component we're looking at the props of - it's the file name).

Reply:
sure thing, if it's not exported Props will do just fine.